### PR TITLE
AAC-680 search bar layout

### PR DESCRIPTION
**Pull request by nilswindisch**

### CODE REVIEW

**Comment by nilswindisch** [comment]
Whatever I do, no option of adding the app.min.css results in a mergable PR. With or without map, compressed or expanded format makes no difference

File: Resources/Private/Scss/extra.scss [hunk 1]
```diff
@@ -0,0 +1,114 @@
+$grid-s-width: 312px;
+$grid-m-width: 768px;
+$grid-l-width: 960px;
+
+$media-s: "(max-width: 767px)";
+$media-m: "(min-width: 768px)";
+$media-l: "(min-width: 1024px)";
```
Comment (snaegle) [inline Resources/Private/Scss/extra.scss]: There seems to be some inconsistency between media-l and grid-l-width. Why is one 1024 and the other 960?

Comment (nilswindisch) [reply Resources/Private/Scss/extra.scss]: It's just the decision when a certain media query triggers a certain grid. could be the same, doesn't have to be. They are the same in your grid system?

Comment (snaegle) [reply Resources/Private/Scss/extra.scss]: I use 768 and 960px, as in https://github.com/subugoe/aac-frontend-docu/blob/master/views_and_grids.md described. 
For the rest or the code, L starts with 961px, S and M are the same. 
We should try to use the same breakpoints.

**Review by snaegle — APPROVED** [review]

**Pull request merged**